qiime2/metadata/metadata.py

```diff
@@ -473,10 +473,10 @@ def __ne__(self, other):
         """
         return not (self == other)
 
-    def save(self, filepath):
-        """Save a TSV metadata file.
+    def save(self, filepath, ext=None):
```

Comment:
Guess what I just found? There's a related class that needs the same "treatment": `MetadataColumn` - you can find it elsewhere in this file. It might be beneficial to move the business logic into `MetadataWriter`.

Reply:
@thermokarst so just to clarify, you want me to move the MetadataColumn class to be housed under the MetadataWriter class within metadata > io.py?

Reply:
Close - I don't think `MetadataColumn` needs to go anywhere, though. You know how over in `results.py` you worked on `Artifact` and `Visualization` by making changes to their parent class, `Result`? I think we actually have something similar here: `Metadata` and `MetadataColumn` are like `Artifact` and `Visualization`. The difference though is that we aren't using direct inheritance - `Metadata` and `MetadataColumn` both just call the `MetadataWriter.write()` method manually. I'm thinking that it might be beneficial to move the "business logic" you have implemented in `Metadata` up into `MetadataWriter.write` - this would be a good use of DRY, IMO. We can chat on a call if you want - you know where to find me.

Reply:
@thermokarst got it, thank you!! Will push up those changes for review right now!